### DISCUSSION TITLE
Problem after update to AIO v2.1.2

### DISCUSSION THREAD

**Discussion by 4tler** [nextcloud/all-in-one#1395] [Questions] [answered]
I would just like to point out that there seems to be a major problem with the shares after updating to NC 24.0.7 which happens inevitably with the AIO update. 

Also under NC 25 the problem seems to exist.

There is a bugreport:  [#34976](https://github.com/nextcloud/server/issues/34976)

**Comment by szaimen** [comment, marked as answer]
Thanks! I pinged the team
